File: dotnet/Tirggers/Models/TodoItem.cs
using System;

namespace Microsoft.Function.Triggers.Models
{
    // SQLTrigger 함수에서 Model 로 사용됨
    public class ToDoItem
    {
        public Guid Id { get; set; }
        public int? order { get; set; }
        public string title { get; set; }
        public string url { get; set; }
        public bool? completed { get; set; }
    }
}

File: python/Triggers/Models/todo_item.py
from dataclasses import dataclass
from typing import Optional
from uuid import UUID


@dataclass
class TodoItem:
    """
    SQL Trigger에서 사용되는 Todo Item 모델
    """
    Id: UUID
    order: Optional[int] = None
    title: Optional[str] = None
    url: Optional[str] = None
    completed: Optional[bool] = None
